packages/react-components/src/templates/NetworkPage.tsx

```diff
@@ -19,10 +19,27 @@ const articleStyles = css({
   padding: `0 0 ${vminLinearCalc(mobileScreen, 36, tabletScreen, 72, 'px')}`,
 });
 
-const NetworkPage: React.FC = ({ children }) => {
+type NetworkPageProps = {
+  searchOnChange: (newQuery: string) => void;
```

Comment:
change handlers in the component lib are always optional (`?:`), initialized to `= noop`. This aids with testing, Storybook, etc., where fewer props need to be provided as a consequence.
Also, naming is usually `onChange*`, so `onChangeToggle`, `onChangeSearch`, ...